activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Allow Range#=== and Range#cover? on Range
+
+    ruby/ruby@989e07c features switching `Range#===` to use internal
+    `r_cover_p` instead of rubyland `include?`. This breaks expected behavior
+    of `ActiveSupport::CoreExt::Range` documented since at least 8b67a02.
```

Comment:
This is excellent detail for anyone interested in why the change was necessary, but the changelog should be a high level description of what changed from the user's perspective - it's probably sufficient to say that `Range#cover?` can now be given a range like `Range#===` and `Range#include?`, and `Range#===` works correctly when given a range on Ruby 2.6.

Reply:
Cleaned up. I hope I did not go too far by also removing the module name.

Reply:
I think that's the right call - users don't interact with the module directly, they just call the methods that it modifies.